Client.xcodeproj/project.pbxproj

```diff
@@ -686,7 +687,7 @@
 		D3FA777B1A43B2990010CD32 /* SearchTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = D3FA777A1A43B2990010CD32 /* SearchTests.swift */; };
 		D3FEC38D1AC4B42F00494F45 /* AutocompleteTextField.swift in Sources */ = {isa = PBXBuildFile; fileRef = D3FEC38C1AC4B42F00494F45 /* AutocompleteTextField.swift */; };
 		D433852C27ABC8150069DD33 /* MappaMundi in Frameworks */ = {isa = PBXBuildFile; productRef = D433852B27ABC8150069DD33 /* MappaMundi */; };
-		D433852D27ABC8250069DD33 /* RUSTPlaces.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 43BE57EA278BA4ED00491291 /* RUSTPlaces.framework */; };
```

Comment:
huh this is not right, probably a side-effect to the merge conflict, I'll fix it

Reply:
Mh 🤔 @electricRGB had the same change in his PR here: https://github.com/mozilla-mobile/firefox-ios/pull/9970
I guess somehow it should never had been there in the first place?

Reply:
Actually, `RUSTPlaces.frameworks` doesn't exist anymore, kinda surprised it's still in the `.pbxproj`

@nbhasin2 (since you have the context about the frameworks and all 😬), **xcode automatically did this for me**, is there an xcode setting to force it to remove the framework from here, or is it safe to let it do this "null" thing it's doing here

Reply:
@lmarceau we commented at the same time 😆 Ya I don't think it should be there

Reply:
@tarikeshaq haha yeah, and same conclusion 😝

Reply:
It is now gone in `main` :)